test/src/sw/redis++/test_main.cpp

```diff
@@ -211,7 +211,7 @@ int getopt(int argc, char **argv, const char *optstring) {
 #endif
 
 void print_help() {
-    std::cerr << "Usage: test_redis++ -h host -p port"
+    std::cerr << "Usage: test_redis++ -h host -p port -u uri"
```

Comment:
As I mentioned in above comment, we cannot support URI for the test problem so far.